db/seeds.rb

```diff
@@ -10,32 +10,23 @@
   Topic.find_or_create_by(title: title)
 end
 
-# user = User.new
-# user.email = 'tiziana.romani@rootstrap.com'
-# user.password = 'aaaaaa'
-# user.password_confirmation = 'aaaaaa'
-# user.gender = 'female'
-# user.full_name = 'Sassss'
-# user.confirm
-# user.save
+# u = User.create!(email: "tiziana.romam@rootstrap.com",
+#   password: 'aaaaaa',
+#   gender:'female',
+# full_name: 'Tiziana')
+# u.confirm
 
-# user = User.new
-# user.email = 'tiziana.romani1@rootstrap.com'
-# user.password = 'aaaaaa'
-# user.password_confirmation = 'aaaaaa'
-# user.gender = 'female'
-# user.full_name = 'Sassss'
-# user.confirm
-# user.save
+# u = User.create!(email: "tiziana.romani1@rootstrap.com",
+#   password: 'aaaaaa',
+#   gender:'female',
+# full_name: 'Tiziana')
+# u.confirm
 
-# user = User.new
-# user.email = 'tiziana.romani2@rootstrap.com'
-# user.password = 'aaaaaa'
-# user.password_confirmation = 'aaaaaa'
-# user.gender = 'female'
-# user.full_name = 'Sassss'
-# user.confirm
-# user.save
+# u = User.create!(email: "tiziana.romani2@rootstrap.com",
+#   password: 'aaaaaa',
+#   gender:'female',
+# full_name: 'Tiziana')
+# u.confirm
```

Comment:
Why the commented code?

Reply:
Because `find_or_create_by` does not work with users when you set password, due to some devise conflict. The solution was to set passwords afterwards:
```
u = User.find_or_create_by(email:"tiziana.romani1@rootstrap.com", gender: 'female', full_name: 'Tiziana')
u.password = 'aaaaaa'
u.save
``` 
I think this is done at some point in the future, I'll write it here anyways